sqlglot/dialects/duckdb.py

```diff
@@ -96,6 +97,21 @@ def _json_format_sql(self: generator.Generator, expression: exp.JSONFormat) -> s
     return f"CAST({sql} AS TEXT)"
 
 
+def _ensure_utf8(charset: exp.Literal) -> None:
+    if charset.name.lower() != "utf-8":
+        raise UnsupportedError(f"Unsupported charset {charset}")
+
+
+def _decode_sql(self: generator.Generator, expression: exp.Decode) -> str:
+    _ensure_utf8(expression.args["charset"])
+    return self.func("DECODE", expression.this, expression.args.get("replace"))
```

Comment:
What is `replace` here? Seems like [duckdb only supports](https://duckdb.org/docs/sql/functions/blob.html) a `blob` argument in `DECODE`?

Reply:
Yes, this was code I copied from the presto dialect a couple of weeks ago to make this work, and I didn't notice that I left replace. Will fix.